docs/pages/guides/local-app-development.mdx

```diff
@@ -71,6 +71,24 @@ If you install [`expo-dev-client`](/develop/development-builds/introduction/#wha
 
 To create a development build, you can use [local app compilation](#local-app-compilation) commands (`npx expo run:[android|ios]`) which will create a debug build and start the development server.
 
+## Local builds using Android product flavors
+
+> **warning** This feature is only available for SDK 52 and above.
+
+If you have a custom Android project with multiple product flavors using different application ids, you can configure `npx expo run:android` to use the correct flavor and build type. Expo supports both `--variant` and `--app-id` to customize the build and launch behavior when building for Android.
+
+The `--variant` flag can be used to switch the Android build type from **debug** to **release**. This flag can also be used to configure a product flavor and build typ, formatted in camelCase. For example, if you have both a **free** and **paid** product flavor, as documented on [Android developers](https://developer.android.com/build/build-variants#change-app-id), you can build a development version of your app with:
+
+<Terminal
+  cmd={['$ npx expo run:android --variant freeDebug', '$ npx expo run:android --variant paidDebug']}
+/>
+
+The `--app-id` flag can be used to launch the app after building using a customized application id. For example, if your product flavor **free** is using `applicationIdSuffix ".free"` or `applicationId "dev.expo.myapp.free"` you can run build and launch the app with:
+
+<Terminal cmd={['$ npx expo run:android --variant freeDebug --app-id dev.expo.myapp.free']} />
+
+> **info** It is also possible to customize the Android build type, but that would break Expo's assumption that the build type **release** is used for production. You might build unoptimized code in your app when using a different build type other than **release** for production.
```

Comment:
```suggestion
> **info** Customizing the Android build type is also possible, but that would break Expo's assumption that the build type **release** is used for production. You might build unoptimized code in your app using a different build type instead of **release**.
```

Let's start with the subject and update the last sentence to avoid repetition of "release for production" since we've stated that release build type is used for production in the first sentence.